.github/workflows/ci.yaml

```diff
@@ -267,26 +253,24 @@ jobs:
         uses: 'google-github-actions/auth@ef5d53e30bbcd8d0836f4288f5e50ff3e086997d'
         with:
           credentials_json: '${{ secrets.GCP_KEY }}'
-      - name: Run Integration Tests
-        shell: bash -l {0}
-        run: |
-          flit install -s
-          pytest -n auto --cov=datajudge --cov-report=xml --cov-append --backend=bigquery tests/integration
-      - name: Generate code coverage report
-        uses: codecov/codecov-action@v3.1.1
+      - name: Run tests
+        uses: ./.github/actions/pytest
         with:
-          file: ./coverage.xml
-
+          backend: bigquery
+          args: tests/integration
```

Comment:
This should have the `-n auto` argument to `pytest`, if not the test just takes ages to run.